epilogue-runtime/src/main/java/edu/wpi/first/epilogue/logging/EpilogueBackend.java

```diff
@@ -200,7 +200,7 @@ default <S> void log(String identifier, Collection<S> value, Struct<S> struct) {
    * @param value the new value of the data field
    */
   default void log(String identifier, Measure<?> value) {
-    log(identifier, value.baseUnitMagnitude());
+    log(identifier + " (" + value.baseUnit().symbol() + ")", value.baseUnitMagnitude());
```

Comment:
Using `log(identifier, value, value.unit().baseUnit())` would be DRYer, but not a blocker